README.md

```diff
@@ -1,2 +1,10 @@
-# glslpp-rs
-A GLSL preprocessor and lexer in Rust.
+# (glsl)pp-rs
+
+<!-- Some Matrix channel, like Naga's? -->
+<!-- No docs, they are non existent for now [![Docs.rs](https://docs.rs/pp-rs/badge.svg)](https://docs.rs/pp-rs) -->
```

Comment:
I think we should enable this just so that we don't forget in the future, We should also add a warning to the README that this is a project very much in an alpha state and as such things will change and documentation is lacking or nonexistent